src/internal/functions/Update-PuppetModuleFixture.ps1

```diff
@@ -27,9 +27,18 @@ function Update-PuppetModuleFixture {
       $ErrorActionPreference = 'Stop'
       $FixturesFilePath = Resolve-Path -Path (Join-Path $PuppetModuleFolderPath ".fixtures.yml")
       $FixturesYaml = Get-Content -Path $FixturesFilePath -Raw | ConvertFrom-Yaml
-      $FixturesYaml.fixtures.forge_modules = @{
-        pwshlib = 'puppetlabs/pwshlib'
+      $FixturesYaml.fixtures.symlinks = @{
+          powershellget = "#{source_dir}"
       }
+      $FixturesYaml.fixtures.repositories = @{
+        pwshlib = @{
+          repo =  'https://github.com/michaeltlombardi/ruby-pwsh'
+          ref  =  '2050855d33692a90017859041f8404cf4e21e49c'
```

Comment:
These changes are temporary until puppetlabs/ruby-pwsh#39 is merged; for now we need to point at the repo and reference. Once that is merged and released we can instead revert to the initial method of pinning to the released module on the forge.